### PR TITLE
Adding 2D strip geometry

### PR DESCRIPTION
**Pull request by rennney**
Adding generator to make 2D strip geometry for the Ew calculation. The generator draws 2 planes coll/ind. There are two supported patterns: 1) with a hole in the middle of the strip 2) holes are on each side of the strip

conf parameter in the generator is not yet utilized. Thus, all of the parameters are hardcoded and code works with only provided domain

### CODE REVIEW

File: pochoir/gen_pcb_2Dstrips.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    barr = numpy.zeros(dom.shape)
+    
+    #all hardcoded parameters should be provided via cfg
+    plane = 0;
+    conf=1 # 2d view configuration
+    widthX = 5.2 #strip width in X
+    widthZ = 3.2 # strip width in Z
+    positionZ = 17 # lowes position of the strip in Z in mm
+    holeWidth = 2.5 # holde diameter in the strip
+    Nstrips = 21 # total number of strips
```
Comment (brettviren) [inline pochoir/gen_pcb_2Dstrips.py:71]: These should be provided by cfg.

File: pochoir/gen_pcb_quarter.py [hunk 2]
```diff
@@ -120,18 +120,19 @@ def draw_pcb_plane(shape,arr,z,r1,r2,free,val):
...
+    draw_pcb_plane(shape,arr,0,r1,r2,1,(v_cath,v_cath))
 
 def generator(dom, cfg):
     arr = numpy.zeros(dom.shape)
     barr = numpy.ones(dom.shape)
-    draw_3D_pcb(arr,dom.shape,41,41,2000,-1,-15000,10)
+    draw_3D_pcb(arr,dom.shape,41,41,2000,-1,-15000,100)
     barr[arr == 0] = 0
-    
+    draw_3D_pcb(arr,dom.shape,41,41,2000,0,-15000,100)
```
Comment (brettviren) [inline pochoir/gen_pcb_quarter.py:137]: Also, these "magic" numbers should come from `cfg` and/or `dom`.

**Review by brettviren — CHANGES_REQUESTED** [review]
If it is more convenient to change these after a merge, let me know.  Otherwise, you can push more commits to this PR.

**Comment by rennney** [comment]
I moved all hard-coded parameters to specific JSON files. Examples of such files are added in the test directory.

**Pull request closed without merging**